spritz/app/src/main/java/com/novoda/spritz/sample/MainActivity.java

```diff
@@ -0,0 +1,122 @@
+package com.novoda.spritz.sample;
+
+import android.animation.TimeInterpolator;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.support.v4.view.animation.FastOutSlowInInterpolator;
+import android.support.v4.view.animation.LinearOutSlowInInterpolator;
+import android.support.v7.app.AppCompatActivity;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import com.airbnb.lottie.LottieAnimationView;
+import com.novoda.spritz.Spritz;
+import com.novoda.spritz.SpritzPage;
+
+import java.util.concurrent.TimeUnit;
+
+public class MainActivity extends AppCompatActivity {
+
+    public static final int PAGER_NUM = 3;
```

Comment:
This should be `private` and probably called `PAGES_COUNT`

Reply:
Done in 635fd6158c58fde214d93abb95a699868919a90b.